modin/pandas/groupby.py

```diff
@@ -323,6 +329,13 @@ def boxplot(
         )
 
     def ngroup(self, ascending=True):
+        if self._is_multi_by or isinstance(self._by, pandas.Grouper):
+            ErrorMessage.default_to_pandas(
+                "Gropuby with multiple columns or Grouper object"
+            )
+            return self._df._default_to_pandas(
+                lambda df: df.groupby(by=self._by).ngroup()
+            )
         index = self._index if not self._axis else self._columns
         return (
             pandas.Series(index=index)
```

Comment:
Should we change this to a modin series? perhaps in a different pr?

Reply:
Probably, yes.